package.json

```diff
@@ -34,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "prettier": "2.6.2"
```

Comment:
You installed prettier which is only necessary if you have scripts like "prettier:check" in your package.json. It is unnecessary if you want prettier only to work with the vs code extension.